packages/sit-onyx/playwright.config.ts

```diff
@@ -40,9 +40,5 @@ export default defineConfig({
     },
   },
   /* Configure projects for major browsers */
-  projects: [
-    { name: "chromium", use: { ...devices["Desktop Chrome"] } },
-    { name: "firefox", use: { ...devices["Desktop Firefox"] } },
-    { name: "webkit", use: { ...devices["Desktop Safari"] } },
-  ],
+  projects: [{ name: "chromium", use: { ...devices["Desktop Chrome"] } }],
```

Comment:
Why did you remove firefox and webkit? Is this unintentionally?

Reply:
Yes, whoops 🙈